pkg/api/types.go

```diff
@@ -388,8 +391,21 @@ const (
 
 	// MountOptionAnnotation defines mount option annotation used in PVs
 	MountOptionAnnotation = "volume.beta.kubernetes.io/mount-options"
+
+	// StorageTopologyConstraint defines topology constraints of a PV
+	// Value is type []TopologyConstraint
+	AlphaStorageTopologyConstraintsAnnotation = "volume.alpha.kubernetes.io/topology-constraints"
 )
 
+// TopologyConstraint contains label information that the resource is constrained to
```

Comment:
```
TopologyConstraint represents topology dimensions for a resource. It consists of a `Topology key` that represents a unique topology dimension and one or more `optional values` that further constrain a specific topology dimension.
Example: Topology key `kubernetes.io/zone` can have values `foo` & `bar` where the resource that this topologyconstraint applies to is accessible only from zones foo and bar.
``` 

This is no longer a traditional `label`. Maybe say `TopologyConstaints are expected to be similar to labels. They are not managed by the system`?